subsys/esb/esb.c

```diff
@@ -168,6 +178,10 @@ static struct payload_rx_fifo rx_fifo;
 static uint8_t tx_payload_buffer[CONFIG_ESB_MAX_PAYLOAD_LENGTH + 2];
 static uint8_t rx_payload_buffer[CONFIG_ESB_MAX_PAYLOAD_LENGTH + 2];
 
+/* Random access buffer variables for ACK payload handling */
+struct payload_random_access_buf_wrapper ack_pl_container[CONFIG_ESB_TX_FIFO_SIZE];
+struct payload_random_access_buf_wrapper * ack_pl_container_entry_point_pr_pipe[CONFIG_ESB_PIPE_COUNT];
```

Comment:
You could rename this local `struct payload_random_access_buf_wrappe` to `pra_buf_wrapper` and  use `ack_pl[]` and `ack_pl_entry`. That would significantly mitigate the unreadability here.